integration-cli/docker_cli_build_test.go

```diff
@@ -1568,7 +1569,18 @@ func TestBuildAddBadLinks(t *testing.T) {
 	}
 	defer os.RemoveAll(tempDir)
 
-	symlinkTarget := fmt.Sprintf("/../../../../../../../../../../../..%s", tempDir)
+	var symlinkTarget string
+	if runtime.GOOS == "windows" {
+		var driveLetter string
+		if abs, err := filepath.Abs(tempDir); err != nil {
+			t.Fatal(err)
+		} else {
+			driveLetter = abs[:1]
+		}
+		symlinkTarget = fmt.Sprintf(`%s:\..\..\..\..\..\..\..\..\..\..\..\..%s`, driveLetter, tempDir)
```

Comment:
@icecrime nice catch. I pushed to fix to chop the drive section off of `tempDir`. I now get `C:\..\..\..\..\..\..\..\..\..\..\..\..\Users\ahmetb\AppData\Local\Temp\test-link-absolute-temp-198842290` for this variable.